packages/api/src/functions/dbAuth/DbAuthHandler.ts

```diff
@@ -149,20 +160,20 @@ type Params = {
   username?: string
   password?: string
   method: AuthMethodNames
-  [key: string]: unknown
+  [key: string]: any
 }
 
-export class DbAuthHandler {
+export class DbAuthHandler<TUser extends Record<string | number, any>> {
```

Comment:
This will severely degrade the experience, as soon as you try to access any property of the user inside the handlers.

Using unknowns here forces people to either specify the generic. There's no reason for us to be so prescriptive here. If they want additional typesafety, they can pass the generic. If they don't want that level of typesafety, they leave it as is.

It's important that we let the user choose the level of complexity they want to introduce in their code with TS, and they have the option to - rather than fight the framework.